doc/content/manuals/theory/index.md

```diff
@@ -167,39 +242,84 @@ The I-soil material model is a nonlinear hysteretic soil model that is based on
 
 !media media/yield_surface.png width=40% margin-left=200px float=center id=fig:yield_surface caption=Invariant yield surfaces of the individual elastic-perfectly curves.
+=======
+### Nonlinear hysteretic constitutive model for soils (I-soil)
+
+I-soil (\citet{numanoglu2017phd}) is a three dimensional, physically motivated, piecewise linearized nonlinear hysteretic material model for soils. The model can be represented by shear type parallel-series distributed nested components (springs and sliders) in one dimensional shear stress space and its framework is analogous to the distributed element modeling concept developed by \citet{iwan1967on}. The model behavior is obtained by superimposing the stress-strain response of nested components. Three dimensional generalization follows \citet{chiang1994anew} and uses von Mises (independent of effective mean stress) and/or Drucker-Prager (effective mean stress dependent) type shear yield surfaces depending on user's choice. The yield surfaces are invariant in the stress space \ref{fig:yield_surface} . Thus, the model does not require kinematic hardening rule to model un/reloading stress-strain response and preserves mathematical simplicity.
 
-The backbone stress-strain curves for each soil layer can be provided in three different ways:
+!media media/theory/yield_surface.png width=40% margin-left=200px float=center id=fig:yield_surface caption=Invariant yield surfaces of the individual elastic-perfectly curves (after Chiang and Beck, 1994).
 
-1. User-defined backbone curve (soil_type = 0): The backbone curve can be provided in a .csv file where the first column is shear strain and the second column is shear stress. The number of elastic-perfectly plastic curves that will be generated from this backbone curve depends on the number of entries in the data file. When many soil layers are present, a vector of data files can be provided as input. The size of this vector should equal the number of soil layers. Also the number of entries in each data file should be the same.
+The current version of I-soil implemented in MASTODON utilizes Masing type un/reloading behavior and is analogous to MAT79 (LS-DYNA) material model but does not exhibit numerical instability observed in MAT79 (\citet{numanoglu2017conf}). Masing type un/reloading is inherently achieved by the model because upon un/reloading the yielded nested components regain stiffness and strength. The cyclic response obtained from current version of the model is presented in \ref{fig:1D_isoil_representation}. Reduction factor type modification on un/reloading behavior (\citet{phillips2009damping}; \citet{numanoglu2017nonmasing}) is an ongoing study within MASTODON framework.
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```

Comment:
This newer version should be kept and the older version from de2079b461234601978c3cd2e70cb5c6fbcfd7f3 should be removed.